Cargo.toml

```diff
@@ -11,7 +11,7 @@ members = [
   "sql-to-dbsp-compiler/lib/sqllib",
 ]
 exclude = [
-  "sql-to-dbsp-compiler/temp",
+  "sql-to-dbsp-compiler",
```

Comment:
Is this change intentional?

Reply:
yes, I am adding some more directories with experiments